### PR TITLE
support mutilarch image

### PR DESCRIPTION
**Pull request by yanmxa**
**What this PR does / why we need it**:
ibm-healthcheck-operator multiarch build for prow.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes [#37022](https://github.ibm.com/IBMPrivateCloud/roadmap/issues/37022)

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```
NONE
```

### CODE REVIEW

**Comment by yanmxa** [comment]
## build images in different arches
1. amd64
```
> pwd                                                                                                                       ~/ibm-healthcheck-operator(feature-multiarch✗)@master001
/root/ibm-healthcheck-operator
> make build-image                                                                                                          ~/ibm-healthcheck-operator(feature-multiarch✗)@master001
Building the ibm-healthcheck-operator binary for amd64...
...
Successfully built 2358454db7f9
Successfully tagged quay.io/opencloudio/ibm-healthcheck-operator-amd64:v20200529-d449bc1
```
2. ppc64le
```
[root@hq-ppc64le1 ibm-healthcheck-operator]# go version
go version go1.13.4 linux/ppc64le
[root@hq-ppc64le1 ibm-healthcheck-operator]# make build-image
Building the ibm-healthcheck-operator binary for ppc64le...
...
STEP 14: COMMIT quay.io/opencloudio/ibm-healthcheck-operator-ppc64le:v20200529-d449bc1
519d8d51ac4b8e7efa24f8e400b249edbe13cee7c1fbc126f207f030db6bea54

[root@hq-ppc64le1 ibm-healthcheck-operator]# docker images
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.
REPOSITORY                                                                                         TAG                                                                       IMAGE ID       CREATED          SIZE
quay.io/opencloudio/ibm-healthcheck-operator-ppc64le                                               v20200529-d449bc1                                                         519d8d51ac4b   26 seconds ago   221 MB
```

File: build/Dockerfile [hunk 1]
```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-FROM registry.access.redhat.com/ubi8/ubi-minimal@sha256:fc7fc7bc6b293fd2ec1c1b035acc6b97ba34f0700c5366b699c646d947c39f81
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
```
Comment (haoqing0110) [inline build/Dockerfile:17]: http://prow.purple-chesterfield.com/log?job=check_ibm-healthcheck-operator&id=1266263169051922434

**Comment by haoqing0110** [comment]
@yanmxa the check_ibm-healthcheck-operator job failed. May need to update file https://github.com/IBM/test-infra/blob/master/prow/cluster/jobs/IBM/ibm-healthcheck-operator/IBM.ibm-healthcheck-operator.master.yaml first before merging this PR.

**Comment by yanmxa** [comment]
https://github.com/IBM/test-infra/pull/176

**Comment by yanmxa** [comment]
> @yanmxa the check_ibm-healthcheck-operator job failed. May need to update file https://github.com/IBM/test-infra/blob/master/prow/cluster/jobs/IBM/ibm-healthcheck-operator/IBM.ibm-healthcheck-operator.master.yaml first before merging this PR.

https://github.com/IBM/test-infra/pull/176

**Pull request merged**